### PR TITLE
New Togglable Settings To Change Image Position and Reset Timestamp on Project Change.

### PR DESCRIPTION
**Pull request by ZanePereira**
[Link To Preview Image](https://fucked-up.party/t3Z563z1.png)

### CODE REVIEW

**Comment by Swan** [comment]
If you're wanting to add this, there needs to be a config option that allows users to switch between large and small images. You also will need to provide the images you are using, so I can update the application.

**Comment by ZanePereira** [comment]
@Swan I have added a togglable option to display the file type as the large image, this seems to work fine. If you do need me to i can change the icons.

File: discord-rpc-vs/Config/Configuration.cs [hunk 1]
```diff
@@ -35,7 +35,10 @@ internal class Configuration
         ///     Dictates whether or not to reset the timestamp
         /// </summary>
         public bool ResetTimestamp { get; set; } = true;
-
+        /// <summary>
+        ///     Dictates whether or not to use the file type large image.
+        /// </summary>
+        public bool DisplayFileTypeImage { get; set; } = true;
```
Comment (Swan) [inline discord-rpc-vs/Config/Configuration.cs]: This should preferably be called `DisplayFileTypeAsLargeImage`.

Comment (Swan) [inline discord-rpc-vs/Config/Configuration.cs]: This should preferably be called `DisplayFileTypeAsLargeImage`.

**Comment by ZanePereira** [comment]
Sure, I'll also make it display above Display Rich Presence in Visual Studio.

**Comment by ZanePereira** [comment]
Btw, i just want to let you know that i am fairly new to c#. Also, im sorry that i took a while to make these changes, i was working on a personal project.

**Comment by ZanePereira** [comment]
I am now working on an Option to reset the timestamp on project change.

**Comment by Swan** [comment]
Everything looks good. Great contribution.

**Pull request merged**